src/main/java/com/gamedoora/backend/userservices/api/UserServicesController.java

```diff
@@ -94,6 +94,13 @@ public ResponseEntity<UserDTO> getUserByEmail(@RequestParam(required = true) Str
     return createResponse(getUserServicesAssembler().getUserByEmail(email), HttpStatus.OK);
   }
 
+  @GetMapping(
+          value = "/studios/emailId",
```

Comment:
Email Id can have characters that can interfere with path. QueryParam would be a better option.